Monika After Story/game/zz_selector.rpy

```diff
@@ -1766,12 +1764,39 @@ init -10 python in mas_selspr:
         # Get the screen to pass events into
         scr = renpy.get_screen("mas_selector_sidebar")
         if scr is not None:
+            scr.scope["mailbox"].search_text = search_query
             # Search
-            flt_items = _selector_search_items(scr.scope["items"], search_query)
-            scr.scope["flt_items"] = flt_items if flt_items is not None else scr.scope["items"]
+            flt_items = _selector_search_items(scr.scope["filtered_items"], search_query)
+            scr.scope["flt_items"] = flt_items if flt_items is not None else scr.scope["filtered_items"]
         # Update the screen
         renpy.restart_interaction()
 
+
+    def mas_item_name_format(item_name):
+        """
+        Formats acs name to be sentence case, with spaces, and pluralized
+
+        IN: 
+            item_name - the text to be formatted
+        OUT:
+            item_name - formatted
+        """
+
+        # excpetions that include a "-" in the name
+        name_space_exceptions = {
+            "s-type-ribbon": "S-type Ribbon"
```

Comment:
I don't like that we will have to maintain this list in the future. Plus custom user acs types won't be in this list anyway. Maybe we should just show types as they are?

Reply:
Showing "-"'s in names would be very untidy, and removing it would make it "s type ribbon" which is wrong, @ThePotatoGuy can you make a decision on this? I think the better option is to add a new exprop for cateogory, anything without it would be uncatorgorised, also the name would be what is shown in the menu then.

Reply:
I think both of those variants are better than having to remember that we should maintain a map that's local to a function scope.

Reply:
I will remove the exception list then, no worries for me, but i feel that somone needs to look at adding new property which could apply to clothes and hair too, as this code can work for that, but I feel that is a developer decision, as it would alter all the outifts but would allow beter sections and would be needed for adding support for @multimokia 's AOC/AHC

Reply:
We should keep the exception list. We can add API functions to add mapping exceptions to the list, so user-defined ones can add code to add their name exceptions.

Exprop isn't that great for this since it means every spritepack with that `acs_type` must include an additional element that is basically a copy of the `acs_type`.

Reply:
Ok I will keep the exception list

Reply:
@ThePotatoGuy @Booplicate I can remove it or keep it, there is no issue either way, but it is only if a - is in the name, which is a very small amount of things. I don't know how it would be made into an API or moved scopes, please advise on this, but for now, I will leave it in and make the other changes, the dev team can then decides on that privately and come back to me with a decision on it.

Reply:
@Booplicate 
We have maps for a ton of other things in the mod, another mapping is hardly that much more complicated.
However, I think we can actually do an exprop route if we added it to the ACSTemplate for s-type-ribbons. 
What do you think about that?

Reply:
I think that could work, but we don't have a template for clothes/hair.

Reply:
This hasn't been addressed yet.

Reply:
@Booplicate I was told by @ThePotatoGuy to leave it

Reply:
@Booplicate @ThePotatoGuy has the final say yes? so I would like clear action from them about what to do next

Reply:
I have been waiting almost 30 days for any kind of response to this, while I know this is not that important and people have lives to live, I feel we need an answer on this so we know what to do, ALL other chnages have been actioned.

Reply:
I'm waiting for TPG response. But even if we kept the mapping, it shouldn't be local anyway.

Reply:
Ok, one option would be if we add something like a "cateogory name" exprop, that could be added TO ANYTHING including hair clothes etc. Meaning the search has another thing it can check, and the menus can work with anything?

Reply:
keep the mapping, move it so it's in the `mas_selspr` name space and we'll make adjustments in the future as necessary.

Reply:
I have added this in the new commit